addons/event/tests/test_event_internals.py

```diff
@@ -40,14 +49,25 @@ def test_event_date_computation(self):
         event.date_begin = datetime(2020, 3, 1, 10, 0, 0)
         self.assertEqual(registration.get_date_range_str(), u'on Mar 1, 2020, 11:00:00 AM')
 
+        # Is actually 8:30 to 20:00 in Mexico
         event.write({
-            'date_begin': '2019-11-09 14:30:00',
-            'date_end': '2019-11-10 02:00:00',
+            'date_begin': datetime(2020, 1, 31, 14, 30, 0),
```

Comment:
Does it matter if you use the string instead of datetime?
I find the string version a bit more readable actually.

Reply:
colors are better in my linter (it is a real decision maker)